src/main.cpp

```diff
@@ -40,7 +40,7 @@ CBigNum bnProofOfWorkLimit(~uint256(0) >> 20); // "standard" scrypt target limit
 CBigNum bnProofOfStakeLimit(~uint256(0) >> 20);
 CBigNum bnProofOfWorkLimitTestNet(~uint256(0) >> 16);
 
-unsigned int nTargetSpacing = 300;
+unsigned int nTargetSpacing = 30;
```

Comment:
Note for the reviewer, this will not affect MainNet as this is a PoW settings.

Reply:
We will create a separate branch for testnet.